### ISSUE TITLE
Reorganize settings a little to introduce the Traffic view

### ISSUE THREAD

**Issue by MichaelArestad** [Automattic/wp-calypso#11615]
The SEO section, the Analytics section, and the related posts should be combined into one section called Traffic.

Ordering of groups:
```
SEO
Ads
Site stats
Related posts
Google analytics
Page title structure
Website meta
Site verification
```

Please let me know if you have any questions at all. I'm sure some will come up around upgrade banners.

This issue will be tacked by several PRs created as part of #11676

**Comment by tyxla** [comment]
I've started doing some preparations to get this one done:
* #11638 - moves Related Posts in a separate component
* #11634 - using real time Google Analytics module activation state

I also have some questions:
* In the new traffic tab, are we always going to display all notices/banners at the top, even if the corresponding settings card is at the bottom of the section?
* Should we consolidate all upgrade banners somehow, or should we duplicate them and display them above the corresponding settings card (this is somewhat related with the previous question)?
* Should all of this affect WordPress.com sites too, or is it intended just for Jetpack sites?

**Comment by MichaelArestad** [comment]
> Should we consolidate all upgrade banners somehow, or should we duplicate them and display them above the corresponding settings card (this is somewhat related with the previous question)?

I suspect we'll be consolidating banners to match what we're doing in JP, but I'm leaving that up to @richardmuscat 

> Should all of this affect WordPress.com sites too, or is it intended just for Jetpack sites?

It should affect all WordPress.com sites.

**Comment by tyxla** [comment]
Thanks for the replies. I've started working on the new Traffic tab in #11788.

**Comment by danjjohnson** [comment]
Can we add the ability to enable/disable ads? Is that for a different issue?

**Comment by tyxla** [comment]
> Can we add the ability to enable/disable ads? Is that for a different issue?

I'd recommend posting it as a separate issue. This issue will wrap around a lot of PRs anyway.

**Comment by tyxla** [comment]
Traffic tab with SEO and Analytics has already been merged - #11788.

I've also started a PR to move Related Posts in Traffic: #12271.

**Comment by tyxla** [comment]
The last remaining task is to add the Site Stats card, that's being addressed in #12272.
